spec/predicates/spdx.md

```diff
@@ -1,47 +1,69 @@
 # Predicate type: SPDX
 
-Type URI: (tentative) https://spdx.dev/Document
+Type URI: https://spdx.dev/Document
 
-Version: 1.0.0
-
-TODO: Ask SPDX project to choose a URI and to review this spec. Ideally the URI
```

Comment:
Perhaps we should leave the TODO suggesting we get someone from the SPDX community to review the spec?